marlowe-apps/src/Language/Marlowe/Runtime/App/List.hs

```diff
@@ -119,7 +90,7 @@ getContract contractId' =
                              $ Right (create, previous)
       }
   in
-    runMarloweSyncClient runHistorySyncClient
+    runMarloweSyncClient runSyncSyncClient
```

Comment:
Not requesting a change for this PR here - just a note that we should rethink the naming of these functions, which are a bit confusing. The names of things are starting to trip over each other.